README.md

```diff
@@ -2,8 +2,132 @@
 
 Mopro is a toolkit for ZK app development on mobile. Mopro makes client-side proving on mobile simple.
 
-For an overview of Mopro, please see [overview](https://zkmopro.org/docs/intro).
+## Getting started
 
-To get started, please see [getting started](https://zkmopro.org/docs/getting-started).
+Clone the repo and run `cargo build` or `cargo test`.
+
+## Pre-built example
+
+There are pre-built examples projects for iOS and Android [here](https://github.com/vimwitch/mopro-app).
+
+## Building an app library
+
+Mopro works by providing a static library and an interface for your app to build proofs. Before you start this tutorial you should have a zkey and wasm file generated by circom.
+
+To get started we'll make a new rust project that builds this library. Run the following commands in your terminal:
+
+```sh
+mkdir mopro-example
+cd mopro-example
+cargo init --lib
+```
+
+This will create a new rust project in the current directory. Now we'll add some dependencies to this project. Edit your `Cargo.toml` so that it looks like the following:
+
+```toml
+[package]
+name = "mopro-example"
+version = "0.1.0"
+edition = "2021"
+
+# We're going to build a static library named mopro_bindings
+# This library name should not be changed
+[lib]
+crate-type = ["lib", "cdylib", "staticlib"]
+name = "mopro_bindings"
+
+# This is a script used to build the iOS static library
+[[bin]]
+name = "ios"
+
+# We're going to build support for circom proofs only
+[features]
+default = ["mopro-ffi/circom"]
+
+[dependencies]
+mopro-ffi = { git = "https://github.com/vimwitch/mopro.git" }
+uniffi = { version = "0.28", features = ["cli"] }
+rust-witness = { git = "https://github.com/vimwitch/rust-witness.git" }
+num-bigint = "0.4.0"
+
+[build-dependencies]
+rust-witness = { git = "https://github.com/vimwitch/rust-witness.git" }
+mopro-ffi = { git = "https://github.com/vimwitch/mopro.git" }
+uniffi = { version = "0.28", features = ["build"] }
+```
+
+Now you should copy your wasm and zkey files somewhere in the project folder. For this tutorial we'll assume you placed them in `test-vectors/circom`.
+
+Now we need to add 4 rust files. First we'll add `build.rs` in the main project folder. This file should contain the following:
```

Comment:
the same as other files
(like template)
but we can easily check if the template works with CI
e.g. download the template and run `cargo run`

and the code can be linked to the template (not sure if it works)
so we don't manually check/maintain the doc

Reply:
Checking if the tutorial works is a good goal, but we should do it in a separate pr.

Reply:
create an issue here
https://github.com/zkmopro/mopro/issues/193